src/typegenPrinter.ts

```diff
@@ -170,7 +170,9 @@ export class TypegenPrinter {
         const importPath = (path.isAbsolute(val.path)
           ? relativePathTo(val.path, outputPath)
           : val.path
-        ).replace(/(\.d)?\.ts/, "");
+        )
+        .replace(/(\.d)?\.ts/, "")
+        .replace(/\\/, "/");
```

Comment:
Can we use `path.normalize`?

Reply:
I tried with `normalize` first but no, because `path.normalize` is platform dependent. There is already the same fix with `replace` in this codebase.

Reply:
Here it is: https://github.com/graphql-nexus/schema/blob/develop/src/typegenAutoConfig.ts#L319

Reply:
We should probably add `g` to the end of the regex here so it replaces all occurances: `/\\/g`

Reply:
Done.